index.d.ts

```diff
@@ -45,5 +45,7 @@ interface Options {
 }
 
 export declare function createEpicMiddleware<T, S>(rootEpic: Epic<T, S>, options?: Options): EpicMiddleware<T, S>;
+export declare function createEpicMiddleware<E, T, S>(rootEpic: E, options?: Options): EpicMiddleware<T, S>;
```

Comment:
Thank you. I didn't know this.
https://github.com/redux-observable/redux-observable/blob/master/src/createEpicMiddleware.js#L34

It's not good overload. I removed it.